content/root/platform-customization.textile

```diff
@@ -120,17 +120,22 @@ Repeat this step for all fallback hosts in the @fallbackHosts@ array, if provide
 
 h3. Modify your code to use your custom environment
 
-Now that you have tested your custom endpoints, you must modify your code to instantiate the Ably client with the provided @ClientOptions@.
+Now that you have tested your custom endpoints, you must modify your code to instantiate the Ably client with the provided @ClientOptions@. For more information about the clientOptions object, see "clientOptions":/api/rest-sdk#client-options.
```

Comment:
We should keep `ClientOptions` consistent here in terms of capitalisation and styled as code. Alternatively, you could just make the existing reference of it into a link which might be simpler / fewer words?

Reply:
👍

Reply:
@m-hulbert, please see: 97a9abcc2f30353a1bd67f63d71582295b7bc787